packages/babel-parser/src/parser/statement.js

```diff
@@ -96,6 +96,81 @@ function babel7CompatTokens(tokens) {
         continue;
       }
     }
+    if (type === tt.templateMiddle || type === tt.templateTail) {
+      if (!process.env.BABEL_8_BREAKING) {
```

Comment:
I think we can swap these two `if`s, since the outer one would be empty with Babel 8.